usmqe/web/tendrl/mainpage/navpage/models.py

```diff
@@ -0,0 +1,63 @@
+"""
+Common page model for navigation bars.
+"""
+
+
+# import pytest
+
+from webstr.core import WebstrModel, By, PageElement
+
+
+# part of url which is placed after 'usm_web_url'
+# for getting on the described page in the module
+LOCATION = '/#'
+
+
+class NavMenuBarsModel(WebstrModel):
+    """
+    Common page model for the main page - navigation.
+    """
+# Coming soon...
```

Comment:
That I don't know, but it has to be done sooner or later. I am not aware about any specification or issue for that.